release 1.3.1

grigorov opened the pull request:
Upgrade helm chart to use 1.3.1(latest release) hcloud-csi-driver. 

I add metadata "namespace"... @invidian why you not use "kube-system" namespace from original deployment yaml?

grigorov committed:
Update to hcloud-csi 1.3.1

invidian commented on templates/hcloud-csi.yml:
```diff
@@ -14,6 +14,7 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: hcloud-volumes
+  namespace: {{ .Release.Namespace }}
```
>I add metadata "namespace"... @invidian why you not use "kube-system" namespace from original deployment yaml?

As far as I know, Helm 3 recommends to not set the namespace explicitly when possible, so created resources will use the namespace for kubeconfig context. This allows to install the chart in any namespace. And there is no specific reason for those resources to be installed in `kube-system` namespace. Any other namespace works just fine for it.

If original code comes with `namespace: kube-system`, I can either remove the line or template it to make it work in other namespace, so removing seems simpler to me, that's why it's removed. Could you please remove it as well?

As a hint for the future, for doing this updates, I use `wget` to write new upstream content into the files and then use `git restore -p` to selectively revert back the templating. This way, it's easy to pick only important upstream changes without touching the templating.

grigorov replied:
I did this.

invidian reviewed (changes requested):
Hey @grigorov, thanks for your PR. Please remove added `namespace` fields, as I mentioned in the review comment. I will test the patch and if it works, I will merge it.

grigorov commented:
@invidian i did changes in PR.

invidian commented on templates/hcloud-csi.yml:
```diff
@@ -103,7 +103,7 @@ spec:
       containers:
         - name: csi-attacher
-          image: quay.io/k8scsi/csi-attacher:v1.2.1
+          image: quay.io/k8scsi/csi-attacher:v2.2.0
```
It seems this version of `csi-attacher` no longer have `--leader-election-type=leases` flag. The pod is now crashing because of it. It needs to be removed.

grigorov replied:
sorry, i check my changes with original file and add all changes. 
And i squash commits.

invidian reviewed (changes requested):
@grigorov I found one broken thing while testing, please take a look.

Could you also submit all patches as a single commit?

invidian committed:
Change default log level from 5 to 4 for controller containers

As they are by default (upstream) pretty chatty and post messages every
few seconds, which are not very useful.

Signed-off-by: Mateusz Gozdek <mgozdekof@gmail.com>

invidian committed:
Chart.yaml: bump version to 0.1.4

Signed-off-by: Mateusz Gozdek <mgozdekof@gmail.com>

invidian reviewed (approved):
Thanks for the PR and changes @grigorov. You removed a bit too much, so to save both of us some time, I pushed some changes, so I can merge it now. You are still the author of original changes :)

I also added some other patch, which I found useful while testing.
